### PR TITLE
Citus Packaging Request Template

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/citus-packaging-request-template.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+name: Citus Packaging Request Template
+about: Citus Packaging Request
```
Comment (onurctirtir) [inline .github/ISSUE_TEMPLATE/citus-packaging-request-template.md]:
```suggestion
name: Citus release checklist template
about: Please make sure to follow the checklist below along the release process.
```

File: .github/ISSUE_TEMPLATE/citus-packaging-request-template.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+title: Bump to Citus X.Y.Z
+labels: release_checklist
```
Comment (onurctirtir) [inline .github/ISSUE_TEMPLATE/citus-packaging-request-template.md]:
```suggestion
labels: release checklist
```

File: .github/ISSUE_TEMPLATE/citus-packaging-request-template.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
+---
+name: Citus Packaging Request Template
+about: Citus Packaging Request
+title: Bump to Citus X.Y.Z
```
Comment (onurctirtir) [inline .github/ISSUE_TEMPLATE/citus-packaging-request-template.md]:
```suggestion
title: Citus release checklist - [vX.Y.Z]
```

**Review by onurctirtir — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/citus-packaging-request-template.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+labels: release checklist
+assignees: gurkanindibay
+
+---
+
+## Several things to note here
+
+These instructions assume you have `$VERSION`, `$PROJECT`, and `$REPO` environment variables set in your shell (e.g. `10.0.2`, `citus`, and `citus`). With those set, code from most steps can be copy-pasted.
+
+**After this checklist, you're still not done: open a release checklist in Enterprise and release there, too!**
```
Comment (hanefi) [inline .github/ISSUE_TEMPLATE/citus-packaging-request-template.md]:
```suggestion
**After this checklist, you're still not done: open a release checklist for Enterprise too!**
```

File: .github/ISSUE_TEMPLATE/citus-packaging-request-template.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+# Update OS Packages
+## Debian and RedHat
+- Change your directory to `packaging` repository directory & checkout `all-$PROJECT` branch.
+- [x] Run the pipeline using branch name as all-citus https://github.com/citusdata/packaging/actions/workflows/update_package_properties.yml. Input tag name and if version is fancy, input the fancy version_no. Other parameters could be kept as is if you want
```
Comment (hanefi) [inline .github/ISSUE_TEMPLATE/citus-packaging-request-template.md]:
```suggestion
- [ ] Run the pipeline using branch name as `all-$PROJECT` https://github.com/citusdata/packaging/actions/workflows/update_package_properties.yml. Input tag name and if version is fancy, input the fancy version_no. Other parameters could be kept as is if you want
```

File: .github/ISSUE_TEMPLATE/citus-packaging-request-template.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  - Then check the following (needed for both debian & redhat):
+    - [x] Updated `pkglatest` variable in the `pkgvars` file to `$VERSION.citus-1`
+  - Then check the following (needed for debian):
+    - [x] A new entry (`$VERSION.citus-1`, `stable`) is added to the `debian/changelog` file
+  - Then check the following (needed for redhat):
+    - [x] `$PROJECT.spec` file is updated:
+      - [x] `Version:` field
```
Comment (hanefi) [inline .github/ISSUE_TEMPLATE/citus-packaging-request-template.md]:
```suggestion
    - [ ] Updated `pkglatest` variable in the `pkgvars` file to `$VERSION.citus-1`
  - Then check the following (needed for debian):
    - [ ] A new entry (`$VERSION.citus-1`, `stable`) is added to the `debian/changelog` file
  - Then check the following (needed for redhat):
    - [ ] `$PROJECT.spec` file is updated:
      - [ ] `Version:` field
```

File: .github/ISSUE_TEMPLATE/citus-packaging-request-template.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+      - [ ] `Source0:` field
+      - [ ] A new entry (`$VERSION.citus-1`) in the `%changelog` section
+- [ ] <s>Check the CI outputs for the PR on packaging repo thoroughly.</s>
+- [ ] Get changes reviewed; merge the PR
+- [ ] Ensure Github Actions builds completed successfully and package count for each os is as below table and packages in postgres versions is compliant with postgres-matrix.yml in the all-citus branch
```
Comment (hanefi) [inline .github/ISSUE_TEMPLATE/citus-packaging-request-template.md]:
```suggestion
- [ ] Ensure Github Actions builds completed successfully and package count for each os is as below table and packages in postgres versions is compliant with `postgres-matrix.yml` in the `all-project` branch
```

File: .github/ISSUE_TEMPLATE/citus-packaging-request-template.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+- x.y
+- x
+- x.y.x-alpine
+- x.y-alpine
+- x-alpine
+- alpine
+
+
+# Update PGXN
+- [ ] Run https://github.com/citusdata/packaging/actions/workflows/update-pgxn-version.yml` on packaging repo, and check the following:
```
Comment (hanefi) [inline .github/ISSUE_TEMPLATE/citus-packaging-request-template.md]:
```suggestion
- [ ] Run https://github.com/citusdata/packaging/actions/workflows/update-pgxn-version.yml on packaging repo, and check the following:
```

**Review by hanefi — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/citus-packaging-request-template.md (hunk 6 above)
Comment (gurkanindibay) [reply .github/ISSUE_TEMPLATE/citus-packaging-request-template.md]: Already fixed :)

**Pull request merged**